src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/use-incident-intelligence.mdx

```diff
@@ -119,28 +97,50 @@ Root cause analysis includes three main UI sections:
 *  **Error logs**: You can explore millions of log messages with a single click and use manual querying to help you find anomalous patterns and hard-to-find problems.
 * **Attributes to investigate**: We scan the distribution of attributes and surface possible causes by finding significant changes in the distribution. For example, for every single transaction event, we can scan to see if an individual user starts to take up an anomalous share of the requests sent to your app. You can also [query interesting attributes](https://docs.newrelic.com/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language). 
 
-## Use decisions [#decisions]
+## Issue timeline [#visual-timeline]
 
-To further reduce noise or get improved incident correlation, you can change or customize your decisions. Decisions determine how Incident Intelligence groups incidents together.
+The visual timeline, as presented below, shows you a breakdown of:
```

Comment:
Should we call it the issue timeline here to match the section title?

Reply:
That's right, thanks!